### DISCUSSION TITLE
valueType of Properties in device profiles

### DISCUSSION THREAD

**Discussion by wangqingpen** [edgexfoundry/community#199] [General]
device profile   There are several types of data vlaueType , can you give a document?    ![image] (https://github.com/edgexfoundry/community/assets/72652818/54fe6afc-f430-48fb-ac26-179e7268be65)

**Comment by farshidtz** [comment]
You can find the list of supported types in the documentation: https://docs.edgexfoundry.org/3.0/microservices/device/profile/Ch-DeviceProfile/

**Comment by farshidtz** [comment]
@wangqingpen I've renamed the title from "data" to "valueType of Properties in device profiles". Please consider setting appropriate titles on your discussions.
